fastlane/actions/gitlab_create_project.rb

```diff
@@ -170,7 +187,10 @@ def self.migrate_issues(gitlab_src, gitlab_dst, project_src, project_dst, userma
             description: issue.description,
             assignee_id: assignee_id,
             milestone_id: milestone_id,
-            labels: issue.labels.join(",")
+            labels: issue.labels.join(","),
+            due_date: issue.due_date,
```

Comment:
I can't test this with an actual due-date since our source gitlab is pretty old and does not have a due date for issues yet 😆 
But it's working as expected when the source has no due date and I assume you've tested it with an existing due date, so this should be fine 👍